shell/platform/android/build.gradle

```diff
@@ -63,3 +63,7 @@ android {
     }
 }
 
+dependencies {
+    implementation 'androidx.lifecycle:lifecycle-process:2.2.0'
```

Comment:
This shouldn't be necessary - do the new classes not resolve in android studio for you without it?

If they don't resolve without, I may need to revert https://github.com/flutter/engine/pull/50840, I think something about that PR may have messed up how we resolve the android_embedding_dependencies jars

Reply:
You know, I'm not sure. I think this was auto-added by IntelliJ. Let me try removing and resyncing.

Reply:
It appears to work fine w/o, removing.